contracts/prize-pool/YieldSourcePrizePool.sol

```diff
@@ -38,11 +38,14 @@ contract YieldSourcePrizePool is PrizePool {
         yieldSource = _yieldSource;
 
         // A hack to determine whether it's an actual yield source
-        (bool succeeded, ) = address(_yieldSource).staticcall(
+        (bool succeeded, bytes memory data) = address(_yieldSource).staticcall(
             abi.encodePacked(_yieldSource.depositToken.selector)
         );
+        string memory invalidMessage = "YieldSourcePrizePool/invalid-yield-source";
```

Comment:
A more elegant implementation is available in the swappable yield source: https://github.com/pooltogether/swappable-yield-source/blob/bf943b3818b81d5f5cb9d8ecc6f13ffecd33a1ff/contracts/SwappableYieldSource.sol#L85